main/test/ca/uwaterloo/flix/language/phase/TestNamer.scala

```diff
@@ -658,7 +658,15 @@ class TestNamer extends AnyFunSuite with TestUtils {
     expectError[NameError.DuplicateUpperName](result)
   }
 
-
+  test("RedefineBuiltinName.01") {
```

Comment:
We need a test for each declaration:

- enums
- effects
- traits

and so on